### DISCUSSION TITLE
Welcome to naemon Discussions!

### DISCUSSION THREAD

**Discussion by sni** [naemon/naemon-core#444] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nook24** [comment]
Let me introduce myself. I have been working with monitoring systems for over 13 years, and I've been using Naemon since its first beta release. A few months ago, I was asked if I would like to join the Naemon team, which was a great honor for me.

Despite my limited C skills, I strive to contribute my best by answering questions and performing occasional bug fixes. I am the maintainer of the Statusengine project, which initially began as a drop-in replacement for NDOutils and has evolved into a modern backend for processing and storing monitoring data provided by Naemon.

In my day job, I am involved in the openITCOCKPIT project, where I extensively do PHP and Go coding.
